app/app/BaseUser.php

```diff
@@ -58,6 +59,14 @@ public static function authenticate(string $email, string $password)
 	*/
 	public static function isSignedIn()
 	{
+		if (!Session::has('email'))
```

Comment:
If a malicious user with some web development skills set a cookie named 'email' to another user's email address, will these changes effectively sign them in?

For security reasons, we want the password required before anyone signs in.  Cookies can be set using a web browser's developer tools so they can't be trusted this much.  There are some cookies such as laravel_session which we have to trust more but they're more trustworthy since they're very difficult to guess.